doc/content/examples/example5b.md

```diff
@@ -0,0 +1,69 @@
+
+
+!listing examples/ex05b/beam_constraint.i
+
+The mesh is generated by using [BeamMeshGenerator](source/index.md) which generates a 1D mesh from and input mesh text file as shown in [fig:stickmodel5bnew]. Each row following `# Nodes` corresponds to a node and contains the node id, x, y and z coordinates of the node, and nodesets that this node belongs to. Each row following `# Element Linear` corresponds to one Edge2 element and contains the element id, node ids of the two nodes that this element connects, and the block id that this element belongs to
+
+!listing examples/ex05b/stick_model.txt
+
+!media media/examples/5b/stickmodel5bnew.png
+       style=width:50%;margin-left:200px;float:center;
+       id=fig:stickmodel5bnew
+       caption=1D stick model using BeamMeshGenerator.
+
+The other sections of the input file are explained at [Example 5a](examples/example5a).
+
+## Results
+
+The response spectra at the top of the internal and containment structures thus obtained are compared with the response spectra obtained from input file which imports a file mesh generated using [Cubit](https://cubit.sandia.gov/). The results using both type of meshing methods are in close agreement to each other.
+
+!row!
+
+!col! small=12 medium=6 large=6
+!media media/examples/5b/internalx.png
+       style=width:400px id=fig:internalx
+       caption=Comparison of the Response Spectra at at an elevation of 22 ft at internal structure in X-direction
+!col-end!
+
+!col! small=12 medium=6 large=6
+!media media/examples/5b/internaly.png
+       style=width:400px id=fig:internaly
+       caption=Comparison of the Response Spectra at at an elevation of 22 ft at internal structure in Y-direction
+!col-end!
+
+!row-end!
+
+!row!
+
+!col! small=12 medium=6 large=6
+!media media/examples/5b/internalz.png
+       style=width:400px id=fig:intrnalz
+       caption=Comparison of the Response Spectra at at an elevation of 22 ft at internal structure in Z-direction
+!col-end!
+
+!col! small=12 medium=6 large=6
+!media media/examples/5b/containmentx.png
+       style=width:400px id=fig:containmentx
+       caption=Comparison of the Response Spectra at at an elevation of 61 ft at internal structure in X-direction
+!col-end!
+
+!row-end!
+
+!col! small=12 medium=6 large=6
+!media media/examples/5b/containmenty.png
+       style=width:400px id=fig:containmenty
+       caption=Comparison of the Response Spectra at at an elevation of 61 ft at internal structure in Y-direction
+!col-end!
+
+!col! small=12 medium=6 large=6
+!media media/examples/5b/containmentz.png
+       style=width:400px id=fig:containmentz
+       caption=Comparison of the Response Spectra at at an elevation of 61 ft at internal structure in Z-direction
+!col-end!
+
+!bibtex bibliography
```

Comment:
Somehow the heading, 'References', is centered here. Looks like the figure formatting is messing with it. Please fix.